### ISSUE TITLE
CLIP Model will not load on CPU-Only Pytorch Build

### ISSUE THREAD

**Issue by mmtatum** [triton-inference-server/server#8176]
**Description**
Transformed a CLIP model to JIT, created the config.pbtxt and uploaded results to the model directory. When the model attempts to load I get the following error: UNAVAILABLE: Not found: unable to load shared library: libnvrtc.so.12: cannot open shared object file: No such file or directory

**Triton Information**
Using Triton Release 2.55.0 

Built container using the following command: ./build.py --backend python --backend pytorch --backend tensorflow --backend ensemble --backend identity --backend square --backend repeat --target-platform linux --endpoint http --endpoint grpc --enable-logging --enable-metrics --enable-stats --enable-cpu-metrics --extra-backend-cmake-arg=tensorflow:TRITON_TENSORFLOW_INSTALL_EXTRA_DEPS=ON --image=gpu-base,nvcr.io/nvidia/tritonserver:25.02-py3-min

Note I had to modify --extra-backend-cmake-arg=tensorflow2:TRITON_TENSORFLOW_INSTALL_EXTRA_DEPS=ON to --extra-backend-cmake-arg=tensorflow:TRITON_TENSORFLOW_INSTALL_EXTRA_DEPS=ON else the build would not run. 

Are you using the Triton container or did you build it yourself?
Built the container myself.

**To Reproduce**
Steps to reproduce the behavior.

Describe the models (framework, inputs, outputs), ideally include the model configuration file (if using an ensemble include the model configuration file for that as well).

Model Transformation: 
- Model: openai/clip-vit-base-patch32
- Framework: transformers
- Output Model: Torchscript

Transformation Code:
from PIL import Image
import requests

from transformers import CLIPProcessor, CLIPModel
import torch
model = CLIPModel.from_pretrained("openai/clip-vit-base-patch32")
processor = CLIPProcessor.from_pretrained("openai/clip-vit-base-patch32")

url = "http://images.cocodataset.org/val2017/000000039769.jpg"
image = Image.open(requests.get(url, stream=True).raw)

inputs = processor(text=["a photo of a cat", "a photo of a dog"], images=image, return_tensors="pt", padding=True)
example_text_input = inputs["input_ids"]
example_text_input = torch.nn.functional.pad(example_text_input, (0,77-example_text_input.shape[1],0,0))
example_image_input = inputs["pixel_values"]
class ModelWrapper(torch.nn.Module):

    def __init__(self, model):
        super().__init__()
        self.model = model

    def forward(self, input_ids, pixel_values):
        outputs = model(input_ids, pixel_values)
        return torch.softmax(outputs.logits_per_image,1)
    
wrapper = ModelWrapper(model)
script_model = torch.jit.trace(wrapper, example_kwarg_inputs={"input_ids" : example_text_input, "pixel_values" : example_image_input})

Triton Model Config:
name: "clip-vit-base-patch32"
platform: "pytorch_libtorch"
max_batch_size : 32
input [
  {
    name: "input__ids"
    data_type: TYPE_FP32
    dims: [77]
  },
  {
    name: "pixel_values"
    data_type: TYPE_FP32
    dims: [ 3, 224, 224 ]
  }
]
output [
  {
    name: "output__0"
    data_type: TYPE_FP32
    dims: [1]
  }
]

**Expected behavior**
i expected my model to load and be usable in the Triton Inference Server.
